docs/getting-started/devenv/using-visual-studio.md

```diff
@@ -1,13 +1,229 @@
+
+2. Download and install the [Visual Studio official MSI Installer](https://go.microsoft.com/fwlink/?linkid=798481). This will install the .NET Core Tooling Preview 1 for Visual Studio 2015.
+
+3. Download and install [NuGet Manager extension for Visual Studio](https://dist.nuget.org/visualstudio-2015-vsix/v3.5.0-beta/NuGet.Tools.vsix). This will install the latest version of the extension.
+
+4. Open Visual Studio, and on the **File** menu, choose **New**, **Project**.
+
+5. In the **New Project** dialog, in the **Templates** list, expand the **Visual C#** node and choose **.NET Core**. You should see three new project templates for **Class Library (.NET Core)**, **Console Application (.NET Core)**, and **ASP.NET Core Web Application (.NET Core)**.
+
+A solution using only .NET Core projects
+----------------------------------------
+
+### Writing the library
+
+1. In Visual Studio, choose **File**, **New**, **Project**. In the **New Project** dialog, expand the **Visual C#** and **.NET Core** nodes, and choose **Class Library (.NET Core)**. 
+
+2. Name the project "Library" and the solution "Golden". Leave **Create directory for solution** checked. Click **OK**.
+
+3. In Solution Explorer, open the context menu for the **References** node and choose **Manage NuGet Packages**.
+
+4. Choose "nuget.org" as the **Package source**, and choose the **Browse** tab. Check the **Including prerelease** checkbox, and then browse for **Newtonsoft.Json**. Click **Install**. 
+
+5. Open the context menu for the **References** node and choose  **Restore packages**.
+
+6. Rename the `Class1.cs` file to `Thing.cs`. Accept the rename of the class. Remove the constructor and add a method: `public int Get(int number) => Newtonsoft.Json.JsonConvert.DeserializeObject<int>($"{number}");`
+
+7. On the **Build** menu, choose **Build Solution**.
+
+   The solution should build without error.
+
+### Writing the test project
+
+1. In Solution Explorer, open the context menu for the **Solution** node and choose **Add**, **New Solution Folder**. Name the folder "test". 
+   This is only a solution folder, not a physical folder.
+
+2. Open the context menu for the **test** folder and choose **Add**. **New Project**. In the **New Project** dialog, choose **Console Application (.NET Core)**. Name it "TestLibrary" and explicitly put it under the `Golden\test` path. 
+
+   > **Important**
+   >
+   > The project needs to be a console application, not a class library.
+
+3. In the **TestLibrary** project, open the context menu for the **References** node and choose **Add**,  **Reference**. 
+
+4. In the **Reference Manager** dialog, check **Library** under the **Projects**, **Solution** node, and then click **OK**. 
+
+5. In the **TestLibrary** project, open the `project.json` file, and replace `"Library": "1.0.0-*"` with `"Library": {"target": "project", "version": "1.0.0-*"}`. 
```

Comment:
I don't know why, but it doesn't sound unreasonable to me that package would be the default since it's way more common to add a package dependency than a project to project dependency.